shells/pdksh/Portfile

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
```

Comment:
You should not add this modeline because the file's whitespace does not conform to it. If you want to fix the port's whitespace, do so in a second commit that does nothing other than fix the whitespace and add the modeline.

Reply:
I was planning on a fix-it-forward kind of deal, but I understand why not to add this now. Removed.